client/services/courseAPI.ts

```diff
@@ -14,7 +14,17 @@ export const getCourse = createApi({
     getCourse: builder.query({
       query: (courseID) => `course/${courseID}`,
     }),
-  })
+    createCourse: builder.mutation({
+      query: (courseData) => ({
+        url: 'course/',
+        method: 'POST',
+        body: courseData,
+      }),
+    }),
+    getCategory: builder.query({
+      query: () => 'category',
```

Comment:
```suggestion
      query: () => 'category/',
```
Please add slash at the end of the url to avoid 301 

![image](https://github.com/framgia/sph-lms/assets/110364690/fbafc5a2-7573-48de-b10a-c61d29e8d65f)